geonode/contrib/monitoring/frontend/LICENSE

```diff
@@ -0,0 +1,25 @@
+BSD 2-Clause License
```

Comment:
I am not expert with licenses, is this compatible with GPL 3 or later?

Reply:
This is probably a leftover from react app starter. I'm not sure how we can handle this.

Reply:
Ciao Paolo,
this is a leftover; this is work done under contracting from GeoSolutions hence were relinquishing control as part of this process or donating the code back to GeoNode.

Reply:
great! thanks :)